chapters/dynamic_state.adoc

```diff
@@ -9,7 +9,7 @@ ifndef::images[:images: images/]
 
 [NOTE]
 ====
-link:https://docs.vulkan.org/spec/latest/chapters/pipelines.html#pipelines-dynamic-state[Vulkan Spec chapter]
+Additional resources: xref:{chapters}dynamic_state_map.adoc[List of all dynamic states with details]
 ====
 
 == Overview
```

Comment:
In here we already point to `pipelines.html#pipelines-dynamic-state` so no need to have the link twice at the top-ish